python/perspective/setup.py

```diff
@@ -139,13 +139,8 @@ def build_extension_cmake(self, ext):
             "-DPSP_WASM_BUILD=0",
             "-DPSP_PYTHON_BUILD=1",
             "-DPSP_PYTHON_VERSION={}".format(PYTHON_VERSION),
-            "-DPython_ADDITIONAL_VERSIONS={}".format(PYTHON_VERSION),
-            "-DPython_FIND_VERSION={}".format(PYTHON_VERSION),
             "-DPython_EXECUTABLE={}".format(sys.executable).replace("\\", "/"),
-            "-DPYTHON_LIBRARY={}".format(sysconfig.get_config_var("LIBDIR")).replace("\\", "/"),
-            "-DPYTHON_INCLUDE_DIR={}".format(sysconfig.get_config_var("INCLUDEPY")).replace("\\", "/"),
-            "-DPython_ROOT_DIR={}".format(sys.prefix).replace("\\", "/"),
-            "-DPython_ROOT={}".format(sys.prefix).replace("\\", "/"),
+            "-DPYTHON_INCLUDE_DIR={}".format(sysconfig.get_path("include")).replace("\\", "/"),
```

Comment:
this one in particular needs to be tested to not regress https://github.com/finos/perspective/commit/61557c6a28a0c70e00cd4ee9160ef22e65c2ad99